vite.config.mts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    coverage: {
+      exclude: ["*.js", "node_modules"]
```

Comment:
why exclude js files here? was `commitlint.config.js` being counted?

Reply:
Right, it was being shown in tests